core/src/main/java/io/confluent/rest/metrics/MetricsResourceMethodApplicationListener.java

```diff
@@ -236,9 +244,64 @@ public MethodMetrics(ResourceMethod method, PerformanceMetric annotation, Metric
               "The 99th percentile request latency in ms", metricTags), 99));
       this.requestLatencySensor.add(percs);
 
+      this.errorSensorUnknown = metrics.sensor(getName(method, annotation, "errors"));
+      HashMap<String, String> tags = new HashMap<>(metricTags);
+      tags.put(ERROR_CODE_TAG_KEY, "unknown");
+      metricName = new MetricName(getName(method, annotation, "request-error-rate"),
+          metricGrpName,
+          "The average number of requests per second that resulted in HTTP error responses",
+          tags);
+      this.errorSensorUnknown.add(metricName, new Rate());
+
+      this.errorSensor100s = metrics.sensor(getName(method, annotation, "errors"));
+      tags = new HashMap<>(metricTags);
+      tags.put(ERROR_CODE_TAG_KEY, "100");
+      metricName = new MetricName(getName(method, annotation, "request-error-rate"),
+          metricGrpName,
+          "The average number of requests per second that resulted in HTTP error responses",
+          tags);
+      this.errorSensor100s.add(metricName, new Rate());
+
+      this.errorSensor200s = metrics.sensor(getName(method, annotation, "errors"));
+      tags = new HashMap<>(metricTags);
+      tags.put(ERROR_CODE_TAG_KEY, "200");
+      metricName = new MetricName(getName(method, annotation, "request-error-rate"),
+          metricGrpName,
+          "The average number of requests per second that resulted in HTTP error responses",
+          tags);
+      this.errorSensor200s.add(metricName, new Rate());
+
+      this.errorSensor300s = metrics.sensor(getName(method, annotation, "errors"));
+      tags = new HashMap<>(metricTags);
+      tags.put(ERROR_CODE_TAG_KEY, "300");
+      metricName = new MetricName(getName(method, annotation, "request-error-rate"),
+          metricGrpName,
+          "The average number of requests per second that resulted in HTTP error responses",
+          tags);
+      this.errorSensor300s.add(metricName, new Rate());
+
+      this.errorSensor400s = metrics.sensor(getName(method, annotation, "errors"));
+      new HashMap<>(metricTags);
+      tags.put(ERROR_CODE_TAG_KEY, "400");
+      metricName = new MetricName(getName(method, annotation, "request-error-rate"),
+          metricGrpName,
+          "The average number of requests per second that resulted in HTTP error responses",
```

Comment:
I'm not sure how the description is used but I would prefer to update the description for each status code if it's safer to do so.